src/main/java/eiteam/esteemedinnovation/commons/util/ReflectionHelper.java

```diff
@@ -102,7 +102,7 @@ public static void init() {
             merchantField = getField("merchant", "field_147037_w", GuiMerchant.class);
             setAccessible(merchantField);
         } catch (NoClassDefFoundError ignore) {
-            FMLLog.warning("[EI] GuiMerchant class not found. You are probably a server.");
+            EsteemedInnovation.logger.warn("[EI] GuiMerchant class not found. You are probably a server.");
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry